server/sonar-web/src/main/webapp/WEB-INF/app/views/sessions/_form.html.erb

```diff
@@ -7,9 +7,10 @@
       <% auth_providers.each do |provider| %>
         <li>
           <a href="<%= ApplicationController.root_context -%>/sessions/init/<%= provider.getKey().to_s %>"
```

Comment:
No need for <%= ApplicationController.root_context -%> anymore (same for line 13)